vxquery-xtest/src/main/java/org/apache/vxquery/xtest/XTest.java

```diff
@@ -36,7 +36,7 @@
 
     XTest(XTestOptions opts) {
         this.opts = opts;
-        reporters = new ArrayList<ResultReporter>();
+        reporters = new ArrayList<>();
```

Comment:
It's actually less redundant this way, as the type is given in the declaration of reporters.